docs/sdk/features.md

```diff
@@ -114,6 +114,46 @@ The default value will be returned in the following scenarios:
 
 For more information on how the default value is used, see [Variable Defaults](/platform/feature-flags/variables-and-variations/variable-defaults).
 
+## Evaluation Reasons
+
+When a Variable is evaluated in DevCycle, the response includes metadata explaining **why** a specific Variation's value was returned. This is captured in the `eval` object, which helps teams debug and understand feature flag decisions more effectively.
+
+DevCycle extends the [OpenFeature Evaluation Details](https://openfeature.dev/specification/types/#evaluation-details) structure with additional reason types and optional fields for richer context.
+
+### Evaluation Object Format
+
+```json
+"eval": {
+  "reason": "REASON_ENUM",
+  "details": "optional string",
+  "target_id": "optional string"
+}
+```
+
+| Field       | Type   | Description                                                              |
+| ----------- | ------ | ------------------------------------------------------------------------ |
+| `reason`    | string | A required enum value indicating why a specific value was returned.      |
+| `details`   | string | An optional string providing extra context about the evaluation outcome. |
+| `target_id` | string | An optional identifier for the matched targeting rule or audience.       |
+
+### Reason Types
+
+The reason field reflects the primary reason a particular value was served. Here are the possible values:
+
+| Reason            | Description                                                                                                                                        | Examples of `details` Property                                                                                                   |
+| ----------------- | -------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------- |
+| `DEFAULT`         | Returned when the default value was used. This may occur due to a type mismatch, a missing configuration, or an error in evaluation logic.         | - Type mismatch<br/>- User Not Targeted</br>- _(no details if unknown)_                                                          |
```

Comment:
nitpick: capitalization
```suggestion
| `DEFAULT`         | Returned when the default value was used. This may occur due to a type mismatch, a missing configuration, or an error in evaluation logic.         | - Type Mismatch<br/>- User Not Targeted</br>- _(no details if unknown)_                                                          |
```